lib/interviewer/components/SettingsMenu/SettingsMenu.tsx

```diff
@@ -0,0 +1,68 @@
+import { SettingsIcon } from 'lucide-react';
+import {
+  Sheet,
+  SheetContent,
+  SheetDescription,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from '~/components/ui/sheet';
+
+import { actionCreators as deviceSettingsActions } from '~/lib/interviewer/ducks/modules/deviceSettings';
+import { Switch as SwitchUI } from '~/components/ui/switch';
+import { useDispatch, useSelector } from 'react-redux';
+
+type SettingsMenuState = {
+  deviceSettings: {
+    enableExperimentalSounds: boolean;
+  };
+};
```

Comment:
It would be better to convert the device settings selectors to typescript so that these could be inferred

Reply:
[530d195](https://github.com/complexdatacollective/Fresco/pull/51/commits/530d1957f116c22dd7e3be4a9b6e339f2889f9bf) resolved